arcgis-ios-sdk-samples/Maps/Generate offline map (overrides)/OfflineMapParameterOverridesViewController.swift

```diff
@@ -0,0 +1,408 @@
+//
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+// wrapper classes used to pass the model objects around by reference
+private class DoubleObject {
+    var double: Double
+    init(_ double: Double){
+        self.double = double
+    }
+}
+private class BoolObject {
+    var bool: Bool
+    init(_ bool: Bool){
+        self.bool = bool
+    }
+}
+
+class OfflineMapParameterOverridesViewController: UITableViewController {
```

Comment:
I'm curious: why didn't you use static cells? It seems like that would have been easier than designing the table view in code.

Reply:
…because I didn't know that was an option haha. Can you reuse static cells? It's nice to only have two models.